src/plusplus.coffee

```diff
@@ -33,7 +33,9 @@ module.exports = (robot) ->
     # from beginning of line
     ^
     # the thing being upvoted, which is any number of words and spaces
-    ([\s\w'@.-]*)
+    ([\s\w'@.-:]*)
```

Comment:
I can do that, but you already have a gsub handling that scenario: https://github.com/ajacksified/hubot-plusplus/blob/master/src/plusplus.coffee#L50